float32[] coordinates
string observerID
